backend/legacyapi/plan.go

```diff
@@ -82,6 +82,9 @@ const (
 	// FeatureIssueAdvancedSearch supports search issue with advanced filter.
 	FeatureIssueAdvancedSearch FeatureType = "bb.feature.issue-advanced-search"
 
+	// FeatureIssueProjectSetting supports some advanced project settings for issue.
+	FeatureIssueProjectSetting FeatureType = "bb.feature.issue-project-setting"
```

Comment:
This should be put under change workflow

Reply:
Done